reconcile/openshift_base.py

```diff
@@ -104,9 +104,21 @@ class HasOrgAndGithubUsername(Protocol):
     github_username: str
 
 
+class HasOCMap(Protocol):
```

Comment:
Nice idea to keep it backwards-compatible :+1: 

About the naming, what is the status there now? I guess in this case `Has` prefix is ok because we already have an `OCMap` class in this context right? Referring to this discussion https://github.com/app-sre/qontract-reconcile/pull/3154#discussion_r1094138599

Reply:
You are right, it shouldn't use the `Has` prefix. I renamed it to `ClusterMap`.

I want to have an AI, like ChatGPT, just for good naming suggestions 😄